elements/lisk-bft/test/protocol_specs/bft_fork_choice_rules_protocol_specs.spec.ts

```diff
@@ -12,12 +12,13 @@
  * Removal or modification of this copyright notice is prohibited.
  */
 
-import forkChoiceSpecs from '../bft_specs/bft_fork_choice_rules.json';
 import { BFT } from '../../src/bft';
 import { Slots as SlotType } from '../../src/types';
 
 const { Slots } = require('@liskhq/lisk-dpos');
 
+const forkChoiceSpecs = require('../bft_specs/bft_fork_choice_rules.json');
```

Comment:
I think this can be `import` too.
json import is always `import * as forkChoiceSpecs from '...path'`